src/output.jl

```diff
@@ -37,34 +37,50 @@ end
 Output() = Output{Float32}()
 
 """
-    run_id, run_path = get_run_id_path(P::Parameters)
-
-Checks existing `run????` folders in output path to determine a 4-digit `run_id` number
-and creates a new folder `run????` with that `run_id`. Also returns the full path
-`run_path` of that folder. Returns `0,"no runpath"` in the case of no output."""
-function get_run_id_path(P::Parameters)
+    run_id = get_run_id(output, output_path)
 
-    @unpack output,output_path = P
+Checks existing `run-????` folders in output path to determine a 4-digit `run_id` number. 
+"""
+function get_run_id(output, output_path)
 
-    if output
+    if output 
         # pull list of existing run???? folders via readdir
-        pattern = r"run\d\d\d\d"                # run???? in regex
+        pattern = r"run-\d\d\d\d"                # run-???? in regex
         runlist = filter(x->startswith(x,pattern),readdir(output_path))
         runlist = filter(x->endswith(  x,pattern),runlist)
-        existing_runs = [parse(Int,id[4:end]) for id in runlist]
+        existing_runs = [parse(Int,id[5:end]) for id in runlist]
 
         # get the run id from existing folders
         if length(existing_runs) == 0           # if no runfolder exists yet
             run_id = 1                          # start with run0001
         else
             run_id = maximum(existing_runs)+1   # next run gets id +1
```

Comment:
Btw, if someone uses "9999" as run_id then the next run_id would be `10000` which is then converted to `"10000"` which however is ignored for `runlist`, so if "9999" exists all runs after that will overwrite into "run-10000"

Reply:
Yes, I know. But I would be okay with that. I would say for the things that we are doing, that we can trust the users to not mess the naming like this.